{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//span[@data-qa='lufexuloga']\"}\n  (Session info: chrome=124.0.6367.119); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x000000010d681a28 chromedriver + 5216808\n1   chromedriver                        0x000000010d679663 chromedriver + 5183075\n2   chromedriver                        0x000000010d1eb527 chromedriver + 406823\n3   chromedriver                        0x000000010d236ff2 chromedriver + 716786\n4   chromedriver                        0x000000010d237181 chromedriver + 717185\n5   chromedriver                        0x000000010d27b1d4 chromedriver + 995796\n6   chromedriver                        0x000000010d25972d chromedriver + 857901\n7   chromedriver                        0x000000010d2786bf chromedriver + 984767\n8   chromedriver                        0x000000010d2594a3 chromedriver + 857251\n9   chromedriver                        0x000000010d229fe3 chromedriver + 663523\n10  chromedriver                        0x000000010d22a92e chromedriver + 665902\n11  chromedriver                        0x000000010d63f940 chromedriver + 4946240\n12  chromedriver                        0x000000010d6459f4 chromedriver + 4970996\n13  chromedriver                        0x000000010d62043e chromedriver + 4817982\n14  chromedriver                        0x000000010d646509 chromedriver + 4973833\n15  chromedriver                        0x000000010d6106c4 chromedriver + 4753092\n16  chromedriver                        0x000000010d668bb8 chromedriver + 5114808\n17  chromedriver                        0x000000010d668d77 chromedriver + 5115255\n18  chromedriver                        0x000000010d679283 chromedriver + 5182083\n19  libsystem_pthread.dylib             0x00007ff81c3a94e1 _pthread_start + 125\n20  libsystem_pthread.dylib             0x00007ff81c3a4f6b thread_start + 15", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"48b5213bef8a6b488abeb3bb460737de\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 - VWO App Positive Test\")\n    @pytest.mark.positive\n    def test_vwo_login_positive(setup):\n        driver = setup\n        loginPage = LoginPage(driver)\n        loginPage.login_to_vwo(usr=\"py2x@thetestingacademy.com\",pwd=\"Wingify@1234\")\n        time.sleep(5)\n        dashboardPage = DashboardPage(driver)\n        assert \"Dashboard\" in driver.title\n>       assert \"Aman\" in dashboardPage.user_logged_in_text()\n\ntests/vwoLoginTests/test_vwlo_login.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/pageObjects/dashboardPage.py:21: in user_logged_in_text\n    return self.get_user_logged_in().text\ntests/pageObjects/dashboardPage.py:17: in get_user_logged_in\n    return self.driver.find_element(*DashboardPage.user_logged_in)\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:741: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x1110dc3d0>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"no such element\",\"message\":\"no such element: Unable to locate element: {\\...ff81c3a94e1 _pthread_start + 125\\\\n20  libsystem_pthread.dylib             0x00007ff81c3a4f6b thread_start + 15\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//span[@data-qa='lufexuloga']\"}\nE         (Session info: chrome=124.0.6367.119); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nE       Stacktrace:\nE       0   chromedriver                        0x000000010d681a28 chromedriver + 5216808\nE       1   chromedriver                        0x000000010d679663 chromedriver + 5183075\nE       2   chromedriver                        0x000000010d1eb527 chromedriver + 406823\nE       3   chromedriver                        0x000000010d236ff2 chromedriver + 716786\nE       4   chromedriver                        0x000000010d237181 chromedriver + 717185\nE       5   chromedriver                        0x000000010d27b1d4 chromedriver + 995796\nE       6   chromedriver                        0x000000010d25972d chromedriver + 857901\nE       7   chromedriver                        0x000000010d2786bf chromedriver + 984767\nE       8   chromedriver                        0x000000010d2594a3 chromedriver + 857251\nE       9   chromedriver                        0x000000010d229fe3 chromedriver + 663523\nE       10  chromedriver                        0x000000010d22a92e chromedriver + 665902\nE       11  chromedriver                        0x000000010d63f940 chromedriver + 4946240\nE       12  chromedriver                        0x000000010d6459f4 chromedriver + 4970996\nE       13  chromedriver                        0x000000010d62043e chromedriver + 4817982\nE       14  chromedriver                        0x000000010d646509 chromedriver + 4973833\nE       15  chromedriver                        0x000000010d6106c4 chromedriver + 4753092\nE       16  chromedriver                        0x000000010d668bb8 chromedriver + 5114808\nE       17  chromedriver                        0x000000010d668d77 chromedriver + 5115255\nE       18  chromedriver                        0x000000010d679283 chromedriver + 5182083\nE       19  libsystem_pthread.dylib             0x00007ff81c3a94e1 _pthread_start + 125\nE       20  libsystem_pthread.dylib             0x00007ff81c3a4f6b thread_start + 15\n\nvenv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: NoSuchElementException"}, "start": 1715135524227, "stop": 1715135531746, "uuid": "2b459ec1-b7fa-49a3-ad78-9d3e796cd72a", "historyId": "8322718835e047b5ec6e11cbed7d8c47", "testCaseId": "8322718835e047b5ec6e11cbed7d8c47", "fullName": "tests.vwoLoginTests.test_vwlo_login#test_vwo_login_positive", "labels": [{"name": "feature", "value": "TC#1 - VWO App Positive Test"}, {"name": "epic", "value": "VWO Login Test"}, {"name": "tag", "value": "positive"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwlo_login"}, {"name": "host", "value": "MBP-C02DRR20MD6M"}, {"name": "thread", "value": "94436-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwlo_login"}]}